src/Sylius/Behat/Resources/config/services/contexts/ui.xml

```diff
@@ -417,7 +417,10 @@
             <argument type="service" id="sylius.behat.page.shop.order.thank_you" />
             <argument type="service" id="sylius.behat.page.shop.home" />
             <argument type="service" id="sylius.behat.page.shop.account.verify" />
+            <argument type="service" id="sylius.behat.page.shop.account.register.thank_you" />
+            <argument type="service" id="sylius.behat.page.shop.account.dashboard" />
             <argument type="service" id="sylius.behat.element.shop.account.register" />
+            <argument type="service" id="sylius.repository.customer" />
```

Comment:
I am not sure, but what is it?

Reply:
![CleanShot 2023-06-28 at 10 16 44](https://github.com/Sylius/Sylius/assets/80641364/f9d4e92f-a87a-4117-bdb3-e7d2c89defef)
It is used to find a last registered user. This solution is proposed by an original author, maybe we can check whether a “Thank You” message is displayed as in other similar cases. I don't have a strong opinion 🤷🏻.